src/main/resources/templates/settings/avatar/avatar.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <title></title>
+  </head>
+  <body>
+    <th:block th:fragment="section">
+      <div th:replace="~{fragments/section-heading::section-heading(~{::avatar-gravatar-heading-body}, ~{})}">
+        <th:block th:fragment="avatar-gravatar-heading-body">
+          <h2 th:text="#{settings.avatar.gravatarEnabled.activation.title}"></h2>
+        </th:block>
+      </div>
+
+      <div class="row">
+        <aside class="col-md-4 col-md-push-8">
+          <p class="help-block tw-text-sm">
+            <svg th:replace="~{icon/info::svg(className='tw-w-4 tw-h-4')}"></svg>
+            <th:block th:text="#{settings.avatar.gravatarEnabled.activation.description.1}" />
+            <a
+              class="tw-inline-flex tw-items-center"
+              target="_blank"
+              rel="noopener"
+              href="https://urlaubsverwaltung.cloud/hilfe/benutzer/#gibt-es-eine-moeglichkeit-ein-profil-bild-zu-hinterlegen"
```

Comment:
wenn du den Link so setzt, dann darf sich in der Hilfe niemals die Überschrift ändern. Denke das sollten wir konfigurierbar machen oder allgemeiner halten